src/resolver.rs

```diff
@@ -2744,7 +2756,9 @@ fn resolve_package_required_entries(
                     DeltaEdgeOrigin::Unpublished { unpublished_index } => {
                         add_entry(RequiredEntry::Unpublished { unpublished_index })
                     }
-                    DeltaEdgeOrigin::StoredLocalAudit { .. } => {}
+                    DeltaEdgeOrigin::StoredLocalAudit { audit_index, .. } => {
+                        add_entry(RequiredEntry::LocalAudit { audit_index })
```

Comment:
While that's technically true (since that's the only thing that would be a candidate for pruning in the first place), I thought it'd be more intuitive to track all local audits that are used.

Reply:
I didn't track all of them originally because that information exists in `DeltaEdgeOrigin` and `RequiredEntry` only exists for pruning. Doesn't matter a ton though.